### PR TITLE
Bug fixes + UI changes for service-specific pages

### PR DESCRIPTION
**Pull request by aileendds**
## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

This pull request…

1. Fixes Marines link on bottom of front page
2. Adds Navy banner
3. Updates all other service banners to be larger and consistent sizes (see 3rd before and after screenshots)
4. Adds a margin under the banner

For issues #96 and #59

## Testing

To verify the changes proposed in this pull request…

1clone this repo,
git checkout service-specific,
set up development dependencies according to CONTRIBUTING.md,
run bin/rails server, and
load up http://localhost:3000/ in your Web browser of choice
scroll to bottom, click "Marines" -> verify that it goes to Marines tab
click on all other service tabs and check the banner and margin

## Screenshots
Before:

screen shot 2017-09-29 at 3 05 18 pm

screen shot 2017-09-29 at 3 05 24 pm

screen shot 2017-09-29 at 3 05 44 pm
(narrower screen)

After:
screen shot 2017-09-29 at 3 12 13 pm

screen shot 2017-09-29 at 3 07 10 pm

screen shot 2017-09-29 at 3 07 25 pm

### CODE REVIEW

File: app/views/service_specific_information/index.html.erb [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
         <li><a href="#air-force" id="tab-air-force"><span>U.S. </span>Air Force</a></li>
         <li><a href="#navy" id="tab-navy"><span>U.S. </span>Navy</a></li>
-        <li><a href="#marines" id="tab-marine-corps"><span>U.S. </span>Marine Corps</a></li>
+        <li><a href="#marines" id="tab-marines"><span>U.S. </span>Marine Corps</a></li>
         <li><a href="#coast-guard" id="tab-coast-guard"><span>U.S. </span>Coast Guard</a></li>
       </ul>
 
       <div class="usa-grid-full tabs-panel" id="army">
-        <%= image_tag("Service-Army.png") %><br>
+        <%= image_tag("Service-Army.png", :class => "svc-spec-banner") %><br>
```
Comment (jgarber623-gov) [inline app/views/service_specific_information/index.html.erb]: Suggest using the newer Ruby 1.9 hash syntax here:

```erb
<%= image_tag 'Service-Army.png', class: 'svc-spec-banner' %>
```

The main difference being use of `foo: 'bar'`-style syntax vs. `:foo => 'bar'`.

**Comment by jamesatheyDDS** [comment]
@aileendds @lfantano The navy banner is from Navy Supply Systems Command. Perhaps a more general Navy banner would be better?

**Comment by Elliottdotgov** [comment]
maybe pull the banner from here: http://www.navy.mil/ 

cc @aileendds @lfantano 

screen shot 2017-10-24 at 5 03 18 pm

**Comment by aileendds** [comment]
Looks like the service pages are getting a redesign. I think this is fine to ship as is for now, or I can strip out all the changes except the bug fix.

**Comment by jamesatheyDDS** [comment]
@aileendds Given the impending redesign, I suggest creating a new branch from master and applying your bug fix to that, and let this PR wither on the vine.

**Comment by aileendds** [comment]
Ok, closing this.

**Pull request closed without merging**